Model/OrderImporter/AllegroReservation.php

```diff
@@ -111,9 +111,10 @@ public function placeReservation(CheckoutFormInterface $checkoutForm): void
             $this->placeReservationsForSalesEvent->execute(
                 $this->getItemsToSell($checkoutForm, false),
                 $this->getSalesChannel(),
-                $this->getSalesEvent(self::ALLEGRO_EVENT_RESERVATION_PLACED, $checkoutForm->getId()));
+                $this->getSalesEvent(self::ALLEGRO_EVENT_RESERVATION_PLACED, $checkoutForm->getId())
+            );
         } catch (\Exception $e) {
-            throw new \Exception(
+            throw new ReservationPlacingException(
                 "Error while placing reservation for order with id [{$checkoutForm->getId()}]",
                 0,
                 $e
```

Comment:
0 is used as code? 
The code should be unique, to help find a place from where exception was thrown

Reply:
If this code should be unique how can I check if code that I will make up is not used by other Exception? Should I also declare this code in my Exception class as a constant variable?

Reply:
nop, just use current timestamp as a code

Reply:
Done

Reply:
it has to be static value (the time when you wrote the line of code) e.g. 1589537388. Making it dynamic would make it impossible to find the error by code, as every time the code would be different.